Documentation/Signoffs/MasterMCU.md

```diff
@@ -0,0 +1,81 @@
+MASTER MICROCONTROLLER
+------------------------
+### Function of the Subsystem
+The master microcontroller (MMCU) must receive data from four seperate local microcontrollers that interface with the
+sensors in other subsystems. This MMCU will read serial data from each local MCU and then send that data to the PC to
+be processed in the simulation.
+
+### Specifications & Constraints
+- Must connect to no less than four local microcontrollers
+- Must provide real-time data transfer of under 100 milliseconds
+- Shall connect to the PC running the simulation
+- Shall have sufficient GPIO pins for communication protocols
+- Shall be powered by the PC
+- Code must not cause deadlock or inhibit any device(s) from sending/receiving data
+
+The definition of real-time is subjective and therefore can not be directly defined. In the case of this system, real-time was
+based on the average human reaction time of 250 milliseconds [1]. Given that delay being considerably high for a real-time system, 
+the team has decided to cut that number to over 1/2 of its value and target less than 100 milliseconds.
+
+### Wiring Schematic
+
+![image](https://user-images.githubusercontent.com/100802413/202863978-89cc0d7a-06cd-4679-a6c5-b71eb77ff12f.png)
```

Comment:
Also, you will need to show that the microcontroller will actually allow this configuration of usage. Timers often pull double duty in MCUs like this. So, it may not work to connect like this in practice.

Reply:
Are you saying that using a timer interrupt would be a better option? I am slightly confused when you say that timers often pull double duty.

Reply:
No, timers are used for operations like interrupts and communications like SPI and I2c.